modules/Bio/Otter/Lace/OnTheFly/QueryValidator.pm

```diff
@@ -262,41 +266,41 @@ sub _fetch_sequences {
           $self->_add_missing_warning($acc, "unknown accession or illegal evidence type");
         }
 
-#        my ($type, $full) = @{$self->_acc_type_full($acc)};
-#        unless ($type) {
-#            $self->_add_missing_warning($acc => 'illegal evidence type');
-#            next;
-#        }
-
-#        my $info = $cache->feature_accession_info($acc);
-#        unless ($info) {
-#            $self->logger->error("No info for '$acc' - this should not happen");
-#            $self->_add_missing_warning($acc => 'internal error');
-#            next;
-#        }
-
-#        unless ($info->{currency} and $info->{currency} eq 'current') {
-#            $self->_add_missing_warning($acc => 'obsolete SV');
-#            next;
-#        }
-
-#        unless ($info->{sequence}) {
-#            $self->_add_missing_warning($acc => 'no sequence');
-#            next;
-#        }
-
-#        my $seq = Hum::Sequence->new;
-#        $seq->name($full);
-#        $seq->type($type);
-#        $seq->sequence_string($info->{sequence});
-
-#        # Will this ever get hit?
-#        if ($full ne $acc) {
-#            $self->logger->error("_fetch_sequences called with partial acc.sv for '$acc','$full'");
-#            $self->_add_remap_warning($acc => $full);
-#        }
-
-#        push(@{$self->seqs}, $seq);
+        # my ($type, $full) = @{$self->_acc_type_full($acc)};
+        # unless ($type) {
+        #     $self->_add_missing_warning($acc => 'illegal evidence type');
+        #     next;
+        # }
+
+        # my $info = $cache->feature_accession_info($acc);
+        # unless ($info) {
+        #     $self->logger->error("No info for '$acc' - this should not happen");
+        #     $self->_add_missing_warning($acc => 'internal error');
+        #     next;
+        # }
+
+        # unless ($info->{currency} and $info->{currency} eq 'current') {
+        #     $self->_add_missing_warning($acc => 'obsolete SV');
+        #     next;
+        # }
+        #
+        # unless ($info->{sequence}) {
+        #     $self->_add_missing_warning($acc => 'no sequence');
+        #     next;
+        # }
+
+        # my $seq = Hum::Sequence->new;
+        # $seq->name($full);
+        # $seq->type($type);
+        # $seq->sequence_string($info->{sequence});
+        #
+        # # Will this ever get hit?
+        # if ($full ne $acc) {
+        #     $self->logger->error("_fetch_sequences called with partial acc.sv for '$acc','$full'");
+        #     $self->_add_remap_warning($acc => $full);
+        # }
+
+        # push(@{$self->seqs}, $seq);
```

Comment:
These all looks like formatting changes, if there is a functional changes - it is impossible to see. Plese revert formatting changes.

Reply:
I'll change this